packages/kbn-unified-field-list/src/containers/unified_field_list_sidebar/field_list_sidebar.tsx

```diff
@@ -245,6 +230,27 @@ export const UnifiedFieldListSidebarComponent: React.FC<UnifiedFieldListSidebarP
     onOverrideFieldGroupDetails: stateService.creationOptions.onOverrideFieldGroupDetails,
   });
 
+  useEffect(() => {
```

Comment:
node, this code was just moved to have access to the new modified list of allFields